docker-compose.yml

```diff
@@ -92,3 +101,7 @@ services:
         image: redis
         ports:
             - 6379
+
+    cherami:
+        dns_search: .
+        image: datoug/cherami-server
```

Comment:
yes should be. Let me investigate how to get the permission(or do we have any permission control) to publish to Uber